spec/fixtures/sshd_config_sles_12_x86_64

```diff
@@ -0,0 +1,139 @@
+# This file is being maintained by Puppet.
```

Comment:
@ghoneycutt 

Spec test was failing because in manifest (manifests/init.pp:146),  sles_12_x86_64 has "Subsystem sftp /usr/lib/ssh/sftp-server" defined but was using fixture of suse_x86_64 where it uses lib64.

So added this new fixture to match what is defined in manifest (line:146).